src/sentry/options/defaults.py

```diff
@@ -443,6 +443,54 @@
 # Drop delete_old_primary_hash messages for a particular project.
 register("reprocessing2.drop-delete-old-primary-hash", default=[])
 
+# BEGIN PROJECT ABUSE QUOTAS
+
+# Example:
+# >>> org = Organization.objects.get(slug='foo')
+# >>> org.update_option("project-abuse-quota.transaction-limit", 42)
```

Comment:
Note that there's currently a convention that all organization options are prefixed with `"sentry:"`. Fine to keep as-is, since there's no actual reason for this.